### PR TITLE
Add Python Code Block for Claimable Balance

### PR DESCRIPTION
**Pull request by apaldiwal**
Add Python code example for Claimable Balance in Glossary

### CODE REVIEW

**Comment by apaldiwal** [comment]
This will add python code example for Claimable Balance op.

File: content/docs/glossary/claimable-balance.mdx [hunk 1]
```diff
@@ -252,6 +307,28 @@ balances, err := client.ClaimableBalances(sdk.ClaimableBalanceRequest{Claimant:
 check(err)
 balanceId := balances.Embedded.Records[0].BalanceID
 ```
+	
+```python
+# Method 1: Not available in the Python SDK yet.
+
+# Method 2: Not available in the Python SDK yet.
```
Comment (Shaptic) [inline content/docs/glossary/claimable-balance.mdx:314]: Is this true? Method 2 just involves parsing the transaction XDR, which I'm pretty sure is something the Python SDK must be able to do. It should map pretty directly to the other SDKs, something like

```python
tx_result = TransactionResult.from_xdr(tx_response.result_xdr)
op_results = tx_result.result.results
op_result = op_results[0].tr
if op_result.type == OperationType.CreateClaimableBalance:
    id = op_result.create_claimable_balance_result.balance_id
    id_hex = id.to_xdr_bytes().hex()
```

(untested, but you get the idea)

File: content/docs/glossary/claimable-balance.mdx [hunk 2]
```diff
@@ -252,6 +307,28 @@ balances, err := client.ClaimableBalances(sdk.ClaimableBalanceRequest{Claimant:
...
+        .for_claimant(B.public_key)
+        .limit(1)
+        .order(desc = True)
+        .call()
+    )
+except (BadRequestError, BadResponseError) as err:
+    print(f"Claimable balance retrieval failed: {err}")
+
+balance_id = balances["_embedded"]["records"][0]["id"]
+print(f"Balance ID: {balance_id}")
```
Comment (Shaptic) [inline content/docs/glossary/claimable-balance.mdx:330]:
```suggestion
balanceId = balances["_embedded"]["records"][0]["id"]
print(f"Balance ID: {balanceId}")
```

for consistency with other variable names

File: content/docs/glossary/claimable-balance.mdx [hunk 3]
```diff
@@ -293,6 +370,28 @@ txResp, err = client.SubmitTransaction(tx)
 check(err)
 ```
 
+```python
+claimBalance = ClaimClaimableBalance(balance_id = balance_id)
+print(f"{A.public_key} claiming {balance_id}")
```
Comment (Shaptic) [inline content/docs/glossary/claimable-balance.mdx:375]:
```suggestion
claimBalance = ClaimClaimableBalance(balance_id = balanceId)
print(f"{A.public_key} claiming {balanceId}")
```

to be consistent with the previous review comment

**Review by Shaptic — COMMENTED** [review]
Thank you for putting the effort into this! I haven't used the Python SDK much so I don't feel like enough of an authority to approve :heavy_check_mark: this, but I did have one comment re: viability of Method 2.

File: content/docs/glossary/claimable-balance.mdx (hunk 1 above)
Comment (apaldiwal) [reply content/docs/glossary/claimable-balance.mdx:314]: Honestly, I just couldn't find how to do this in the python documentation. I'll try to fix it

Comment (apaldiwal) [reply content/docs/glossary/claimable-balance.mdx]: Thanks for your help! I have made the necessary changes in Method 2.

**Review by Shaptic — APPROVED** [review]
Confirmed that this works, nice work and thank you for contributing! :+1:

**Comment by apaldiwal** [comment]
> Confirmed that this works, nice work and thank you for contributing! 👍

Thank you for approving the PR! :)

**Pull request merged**